gaps/gap-35_gpu_pci_capability/gap-35_gpu_pci_capability.md

```diff
@@ -76,21 +77,21 @@ Namespace that describes information about GPU clocks
 
 | Property                                           | Type      | Applies to | Description                                                                                                        |
 | -------------------------------------------------- | --------- | ---------- | ------------------------------------------------------------------------------------------------------------------ |
-| `golem.!exp.gap-35.v1.inf.gpu.clock.graphics.mhz` | `integer` | Offer      | The max rate of the graphics clock as reported by nvidia-smi                                                       |
-| `golem.!exp.gap-35.v1.inf.gpu.clock.memory.mhz`   | `integer` | Offer      | The max rate of the memory clock as reported by nvidia-smi                                                         |
-| `golem.!exp.gap-35.v1.inf.gpu.clock.sm.mhz`       | `integer` | Offer      | The max rate of the streaming multiprocessor clock as reported by nvidia-smi. CUDA cores are driven by this clock. |
-| `golem.!exp.gap-35.v1.inf.gpu.clock.video.mhz`    | `integer` | Offer      | The max rate of the video clock as reported by nvidia-smi                                                          |
+| `golem.!exp.gap-35.v1.inf.gpu.clocks.graphics.mhz` | `integer` | Offer      | The max rate of the graphics clock as reported by nvidia-smi                                                       |
+| `golem.!exp.gap-35.v1.inf.gpu.clocks.memory.mhz`   | `integer` | Offer      | The max rate of the memory clock as reported by nvidia-smi                                                         |
+| `golem.!exp.gap-35.v1.inf.gpu.clocks.sm.mhz`       | `integer` | Offer      | The max rate of the streaming multiprocessor clock as reported by nvidia-smi. CUDA cores are driven by this clock. |
+| `golem.!exp.gap-35.v1.inf.gpu.clocks.video.mhz`    | `integer` | Offer      | The max rate of the video clock as reported by nvidia-smi                                                          |
```

Comment:
Unification of property. Implementation and a part of this GAP had `clocks` so I updated the name only here.

Reply:
`ya-runtime-vm-nvidia` has `clocks`, so we're good:
https://github.com/golemfactory/ya-runtime-vm-nvidia/blob/257ee60195996bc69f16d3721e5dbea371ab8495/self_test_img/src/lib.rs#L181